### DISCUSSION TITLE
Add a universal APK in the releases?

### DISCUSSION THREAD

**Discussion by KobeW50** [JunkFood02/Seal#1090] [Ideas]
Some users don't know the arch of their device, so it's easier for them to have a universal APK they can download which will certainly work on their device.

I didn't see this requested on GitHub, and a bug report or feature request didn't seem appropriate for this suggestion.

Thank you for considering

**Comment by Marco-9456** [comment]
We used to release Universal versions of Seal, but the size was a problem as far as I remember. Recently, JunkFood, the main developer, came up with a great way to make the app smaller, so I don't think it will be a problem.

**Reply by BobbyESP** [reply]
Yes; the idea is to just leave the core files (that are smaller than 1MB if I remember well; Python and FFMpeg executables) and download from a source to be determinated the Python and FFMpeg packages.
